jenkins/shared-library/vars/configureArtifactRepository.groovy

```diff
@@ -1,25 +1,40 @@
-void call(String environment) {
+void call(String environment, Boolean publicGitHubRepo) {
 	final String ownerName = helper.resolveOrganizationName()
 	logger.logInfo("Configuring respository pull for ${environment}")
 
-	String url = resolveRepositoryUrl(ownerName, resolveRepositoryName(environment))
+	final String url = resolveRepositoryUrl(ownerName, resolveRepositoryName(environment, publicGitHubRepo))
 
 	if (null != url) {
 		configure(environment, ownerName, url)
+
+		if ('javascript' == environment && fileExists('package-lock.json')) {
+			// remove package-lock.json file since the hashes will not match private repository
+			sh('rm -f package-lock.json')
```

Comment:
why this ?

Reply:
When trying to update from internal packages such as symbol-sdk, it gets a hash error due to hash does not match what’s in the package-lock.json file.

Reply:
you mean projects depending on private artifacts, i.e. using symbol-sdk?

Reply:
yea, so the package-lock.json file resolved the package info to npmjs.com with their hash.  I guess this is different from hash in Nexus(due to Metadata).  Removing the package-lock.json will fix the hash error.